route/src/route_generator_worker.cpp

```diff
@@ -373,8 +373,12 @@ namespace route {
             // check if we left the seleted route by cross track error
             if(std::fabs(current_crosstrack_distance_) > cross_track_max_)
```

Comment:
Shouldn't the added functionality replace this behavior? The whole point is to not have a fixed cross track value that would need to change between roads.